### PR TITLE
SR live image changes to run Bsa.efi and Sbsa.efi for SBSA 6.1 coverage

### PR DESCRIPTION
**Pull request by chetan-rathore**
* Defined new sbsa.nsh for SR live image
* fetch and build bsa-acs and linux-acs by default
* Saving output of bsa and sbsa uefi binary to output folder
* Added es_bsa.patch to SR patches folder for bsa uefi build
* Defined sr_flag to run sbsa linux app from init.sh for sr live image

### CODE REVIEW

File: common/config/startup.nsh [hunk 1]
```diff
@@ -66,6 +66,10 @@ endfor
 :DoneDebug
 
 for %j in 0 1 2 3 4 5 6 7 8 9 A B C D E F then
+    if exist FS%j:\EFI\BOOT\bsa\sr_bsa.flag then
+        FS%j:\EFI\BOOT\bsa\sbsa.nsh
+        goto Donebsa
+    endif
     if exist FS%j:\EFI\BOOT\bsa\bsa.nsh then
```
Comment (edhay) [inline common/config/startup.nsh:73]: In the order of running, can we run bsa first and then sbsa?

File: common/scripts/make_image.sh [hunk 2]
```diff
@@ -57,7 +57,11 @@ create_cfgfiles ()
 
     mcopy -i  $fatpart_name -o ${GRUB_FS_CONFIG_FILE} ::/grub.cfg
     mcopy -i  $fatpart_name -o ${EFI_CONFIG_FILE}     ::/EFI/BOOT/startup.nsh
-    mcopy -i  $fatpart_name -o ${BSA_CONFIG_FILE}    ::/EFI/BOOT/bsa/bsa.nsh
+    if [ "$BUILD_PLAT" = "SR" ]; then
+        mcopy -i  $fatpart_name -o ${SBSA_CONFIG_FILE}    ::/EFI/BOOT/bsa/sbsa.nsh
+    else
+        mcopy -i  $fatpart_name -o ${BSA_CONFIG_FILE}    ::/EFI/BOOT/bsa/bsa.nsh
```
Comment (edhay) [inline common/scripts/make_image.sh:63]: Shouldn't bsa.nsh be copied for SR?

Comment (chetan-rathore) [reply common/scripts/make_image.sh:63]: sbsa.nsh will take care of running both BSA and SBSA uefi binaries.

File: common/config/startup.nsh (hunk 1 above)
Comment (chetan-rathore) [reply common/config/startup.nsh:73]: sbsa.nsh runs BSA binary first and then SBSA

**Pull request merged**